src/Sylius/Bundle/UserBundle/Form/Type/CustomerGuestType.php

```diff
@@ -12,33 +12,31 @@
 namespace Sylius\Bundle\UserBundle\Form\Type;
 
 use Sylius\Bundle\ResourceBundle\Form\Type\AbstractResourceType;
-use Sylius\Bundle\UserBundle\Form\EventListener\CustomerRegistrationFormListener;
-use Sylius\Bundle\UserBundle\Form\EventListener\UserRegistrationFormListener;
-use Sylius\Component\Resource\Repository\RepositoryInterface;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
 use Symfony\Component\Form\FormBuilderInterface;
-use Symfony\Component\OptionsResolver\OptionsResolverInterface;
 
 /**
  * @author Michał Marcinkowski <michal.marcinkowski@lakion.com>
  * @author Dmitrijs Balabka <dmitry.balabka@gmail.com>
+ * @author Mateusz Zalewski <mateusz.zalewski@lakion.com>
  */
 class CustomerGuestType extends AbstractResourceType
 {
     /**
-     * @var RepositoryInterface
+     * @var EventSubscriberInterface
      */
-    private $customerRepository;
+    private $eventSubscriber;
```

Comment:
`eventSubscriber`? It doesn't say anything about the object behind this variable. I would call it `guestCustomerSubscriber` or similar.